modules/javafx.web/src/main/native/Source/WebCore/bindings/java/dom3/java/com/sun/webkit/dom/NodeImpl.java

```diff
@@ -38,6 +35,7 @@
 import org.w3c.dom.events.Event;
 import org.w3c.dom.events.EventListener;
 import org.w3c.dom.events.EventTarget;
+import com.sun.webkit.Disposer;
```

Comment:
I raised this question in PR #1420 as well, but didn't follow-up on it. We should make a deliberate decision on this rather than "whatever Andy's version of Eclipse happens to think is a good default". Let's resolve the discussion in _that_ PR and then apply it to all.